kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -262,6 +262,18 @@ define([
 
         /* LISTENERS */
 
+        _encodeChannel(channelType, channelId) {
```

Comment:
changed the way that listeners are stored so that they are indexed by channel type and ID, rather than just by ID -- with the latter method, it is not possible to add both a job and a batch channel listener for the same ID.